com.eviware.soapui.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.eviware.soapui</id>
```

Comment:
`eviware.com` doesn't seem to answer, albeit the domain exist. Given that there is no mention of this in any of the URL I believe it's not the right choice.

Reply:
ok, perfect.
Changed to `org.soapui.SoapUI`.